server/fleet/secrets.go

```diff
@@ -1,3 +1,24 @@
 package fleet
 
-const FLEET_SECRET_PREFIX = "FLEET_SECRET_"
+import (
+	"fmt"
+	"strings"
+)
+
+const ServerSecretPrefix = "FLEET_SECRET_"
+
+type MissingSecretsError struct {
+	MissingSecrets []string
+}
+
+func (e MissingSecretsError) Error() string {
+	secretVars := make([]string, 0, len(e.MissingSecrets))
+	for _, secret := range e.MissingSecrets {
+		secretVars = append(secretVars, fmt.Sprintf("\"$FLEET_SECRET_%s\"", secret))
```

Comment:
```suggestion
		secretVars = append(secretVars, fmt.Sprintf("\"$%s_%s\"", ServerSecretPrefix, secret))
```

Reply:
Ah that last one caused the CI issue, the server name ends in an underscore